frontend/src/metabase/embedding/components/SharingMenu/MenuItems/PublicLinkMenuItem.tsx

```diff
@@ -30,12 +38,21 @@ export function PublicLinkMenuItem({
             t`Create a public link`
           )
         ) : (
-          <Link to="/admin/settings/public-sharing" target="_blank">
-            <Stack gap="xs">
-              <Title order={4}>{t`Public links are off`}</Title>
-              <Text size="sm">{t`Enable them in settings`}</Text>
-            </Stack>
-          </Link>
+          <>
+            {t`Public link`}
+            <Button
+              component={Link}
+              to="/admin/settings/public-sharing"
+              target="_blank"
+              variant="subtle"
+              h="auto"
+              lh="inherit"
+              ml="sm"
+              className={cx(CS.floatRight, CS.borderless, CS.p0)}
```

Comment:
I think we want to avoid pulling in these global classes where possible, and in any event, I don't think `.borderless` or `.floatRight` are doing anything here


```suggestion
              p={0}
```